_typos.toml

```diff
@@ -0,0 +1,23 @@
+[default]
+locale = "en-us"
+
+[files]
+extend-exclude = [
+    # generated files
+    "book/ethicalads-theme.css",
+    "fuzzing/fuzz/artifacts/",
+    "fuzzing/fuzz/corpus/",
+    "target/",
+    # too many false positives
+    "testing/tests/gen_ws_tests.py",
+]
+
+[default.extend-words]
+# French words
+exemple = "exemple"
+existant = "existant"
+# fillter words
+Ba = "Ba"
+fo = "fo"
+Fo = "Fo"
```

Comment:
I never saw these two as fillers.

Reply:
Hm, yes, "fo", "Fo" & "Ba" are actually not used in filler texts, but "foO", "FoO", "BaR", in casing tests. Will change the comment.